### DISCUSSION TITLE
Plugin APIs Not Working

### DISCUSSION THREAD

**Discussion by BridgeCross** [mattieserver/netbox-topology-views#453] [Q&A]
Hi,

Has anyone had success making API calls using the plugin's APIs and the API tokens from Netbox?

I've been trying to work with the APIs but it does not work and returns the login page for my NetBox instance as a response rather than the desired result.

I tried it in the swagger that comes with NetBox and realised that the API tokens I get from NetBox do not properly grant authorization for the plugin's APIs but the tokens work for the other NetBox APIs.

Any help would be appreciated. Thanks.

Netbox Version: 3.6.3
Netbox Topology Views: 3.8.0

**Comment by dreng** [comment]
Hi,

sounds familiar. See #421 and #422. I could not get it to work myself so far. Maybe @mattieserver can chime in here.

**Reply by BridgeCross** [reply]
Thanks for the feedback
